_source/_posts/2021-09-08-sonar-qube-dotnet.md

```diff
@@ -0,0 +1,167 @@
+---
+layout: blog_post
+title: "How to Write Cleaner, Safer Code with SonarQube, Docker and .NET Core"
+Author: [pawel-szydziak]
+by: contractor
+communities: [.net]
+description: "This tutorial demonstrates how to use SonarQube with .NET Core. "
+tags: [sonarqube, csharp-9, dotnet, dotnetcore, csharp, aspnet, aspnetcore, dotnet5]
+tweets:
+- "Learn all you need to know about #sonarqube + #dotnetcore in this new tutorial ->"
+- "Get up to speed on using #sonarqube and #dotnet in this post"
+- "Wondering how #sonarqube works with @oktadev in #dotnet? We've got you covered!"
+Image: blog/featured/okta-dotnet-books-quarter.jpg
+type: awareness
+---
+
+
+{% img blog/reactive-java/project-reactor.png alt:"Project Reactor Logo" width:"300" %}{: .center-image }
```

Comment:
This image doesn't seem like it should be here.

Reply:
```suggestion
```
Correct: It was left behind as a placeholder, removing it now! Hopefully with correct spacing